CONTRIBUTING.md

```diff
@@ -1,193 +1,49 @@
-# Developer Introduction
+# Contributing to ProbNum
```

Comment:
This title is a duplication of the section header in the docs:

![Screenshot from 2021-05-12 17-19-59](https://user-images.githubusercontent.com/29815984/118000703-58925600-b346-11eb-9876-990a3a278f7e.png)

My suggestion would be:
```
Development
---- Contributing to ProbNum
---- Opening a Pull Request
---- Detailed Developer Guides
---- Style Guide
---- Benchmarking Code
```

Reply:
yes, that sounds better. will change

Reply:
I used 'Creating a Pull Request' to be consisted with the rest of the doc.